config/jobs/ppc64le-cloud/docker-in-docker/postsubmit-push-docker-in-docker.yaml

```diff
@@ -0,0 +1,98 @@
+postsubmits:
+  ppc64le-cloud/test-infra:
+    - name: postsubmit-push-dind
+      decorate: true
+      run_if_changed: 'images/docker-in-docker'
+      branches:
+        - master
+      extra_refs:
+        - base_ref: v0.1.2
+          org: vmware-tanzu
+          repo: buildkit-cli-for-kubectl
+      spec:
+        serviceAccountName: build-img
+        containers:
+          - image: quay.io/powercloud/all-in-one:0.2
+            env:
+              - name: DOCKER_CONFIG
+                valueFrom:
+                  secretKeyRef:
+                    name: quay-powercloud-regcred
+                    key: .dockerconfigjson
+            volumeMounts:
+              - name: kubeconfig
+                mountPath: /etc/kubeconfig
+            command:
+              - /bin/bash
+            args:
+              - -c
+              - |
+                set -o errexit
+                set -o nounset
+                set -o pipefail
+                # Build and install vmware-tanzu/buildkit-cli-for-kubectl
+                pushd $GOPATH/src/github.com/vmware-tanzu/buildkit-cli-for-kubectl
+                make build install
+                popd
+                # Install manifest-tool
+                curl -L https://github.com/estesp/manifest-tool/releases/download/v1.0.3/manifest-tool-linux-`go env GOARCH` -o /usr/local/bin/manifest-tool
+                chmod +x /usr/local/bin/manifest-tool
+                # Kubeconfig for the x86 cluster
+                ca=$(base64 -w 0 /run/secrets/kubernetes.io/serviceaccount/ca.crt)
+                token=$(cat /run/secrets/kubernetes.io/serviceaccount/token)
+                namespace=$(cat /run/secrets/kubernetes.io/serviceaccount/namespace)
+                cat >kubeconfig <<EOL
+                apiVersion: v1
+                kind: Config
+                clusters:
+                - name: default-cluster
+                  cluster:
+                    certificate-authority-data: ${ca}
+                    server: https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT_HTTPS}
+                contexts:
+                - name: default-context
+                  context:
+                    cluster: default-cluster
+                    namespace: ${namespace}
+                    user: default-user
+                current-context: default-context
+                users:
+                - name: default-user
+                  user:
+                    token: ${token}
+                EOL
+                export KUBECONFIG=$(pwd)/kubeconfig
+                # Get the latest tag in the quay.io/powercloud/docker-ce-build
+                latest_tag=$(curl -H "Authorization: Bearer XYZ" -X GET "https://quay.io/api/v1/repository/powercloud/docker-ce-build/tag/" | jq .tags[].name | grep v | head -1 | sed -e 's|\"||g')
```

Comment:
@florencepascual Will the latest_tag remain as 0. every time we push an image.
How about having the tags match the corresponding version of official docker-in-docker image?

Reply:
I am not sure I get what you mean. This docker image is not supposed to be an official docker in docker image. It enables a docker in docker but contains some more packages that are not in the official docker in docker image, such as s3fs and go among others